### PR TITLE
Add new function for take a substring of html result

### PR DESCRIPTION
**Pull request by TrustedSkuola**

### CODE REVIEW

File: src/Converter.php [hunk 1]
```diff
@@ -29,6 +29,25 @@ public function getAsText(): string
         return $output;
     }
 
+    public function getSubstringHtml(string $numberOfCharacter, bool $escape = false): string
+    {
+        $output = '';
+        $checkOutput = '';
+        
+        foreach ($this->rows as $row) {
+            $checkOutput .= $row.PHP_EOL;
```
Comment (delda) [inline src/Converter.php]: Why do you use `PHP_EOL`? It's a useless character to count. What do you think about?

File: src/Converter.php [hunk 2]
```diff
@@ -29,6 +29,25 @@ public function getAsText(): string
         return $output;
     }
 
+    public function getSubstringHtml(string $numberOfCharacter, bool $escape = false): string
```
Comment (delda) [inline src/Converter.php]:
```suggestion
    public function getSubstringHtml(string $numberOfCharacters, bool $escape = false): string
```

File: src/Converter.php [hunk 3]
```diff
@@ -29,6 +29,25 @@ public function getAsText(): string
...
+    {
+        $output = '';
+        $checkOutput = '';
+        
+        foreach ($this->rows as $row) {
+            $checkOutput .= $row.PHP_EOL;
+
+            if ( strlen($checkOutput) > $numberOfCharacter ){
```
Comment (delda) [inline src/Converter.php]:
```suggestion
            if ( strlen($checkOutput) > $numberOfCharacters ){
```

File: src/Converter.php (hunk 1 above)
Comment (TrustedSkuola) [reply src/Converter.php]: Yeah it's typo from a previous version of the code, i'll remove in next commit :)

File: src/Converter.php [hunk 4]
```diff
@@ -29,6 +29,25 @@ public function getAsText(): string
         return $output;
     }
 
+    public function getSubstringHtml(string $numberOfCharacters, bool $escape = false): string
```
Comment (garak) [inline src/Converter.php]: Why `$numberOfCharacters` is a string?

Comment (TrustedSkuola) [reply src/Converter.php]: Because i'm an idiot. I'll change to a int

File: src/Converter.php [hunk 5]
```diff
@@ -29,6 +29,25 @@ public function getAsText(): string
...
 
+    public function getSubstringHtml(int $numberOfCharacters, bool $escape = false): string
+    {
+        $output = '';
+        $checkOutput = '';
+        
+        foreach ($this->rows as $row) {
+            $checkOutput .= $row;
+
+            if ( strlen($checkOutput) > $numberOfCharacters ){
```
Comment (garak) [inline src/Converter.php]: These random spaces make me guess that you forgot to run php-cs-fixer...

Comment (TrustedSkuola) [reply src/Converter.php]: Yep i've ran cs-fixer

**Pull request merged**